cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebViewHelper.java

```diff
@@ -95,6 +102,89 @@ public void run() {
         });
     }
 
+    public static void setBackgroundTransparent(final int index) {
+        if(android.os.Build.VERSION.SDK_INT >10) {
+            sCocos2dxActivity.runOnUiThread(new Runnable() {
+                @Override
+                public void run() {
+                    Cocos2dxWebView webView = webViews.get(index);
+                    if (webView != null) {
+                        webView.setBackgroundColor(Color.TRANSPARENT);
+                        try {
+                            Method method = webView.getClass().getMethod("setLayerType",int.class,Paint.class);
+                            method.invoke(webView,WebView.LAYER_TYPE_SOFTWARE,null);
+                        } catch (NoSuchMethodException e) {
+                            e.printStackTrace();
+                        } catch (IllegalAccessException e) {
+                            e.printStackTrace();
+                        } catch (InvocationTargetException e) {
```

Comment:
Can we just catch exception in one place if there is not specific operation for exceptions, such as 
```java
catch (Exception e) {
    e.printStackTrace();
}
```

And since which API level has `setLayerType `? I can not find the information [here](https://developer.android.com/reference/android/webkit/WebView.html#setLayerType(int, android.graphics.Paint)).